db/migrate/20241111133654_create_discourse_activity_pub_clients.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class CreateDiscourseActivityPubClients < ActiveRecord::Migration[7.1]
+  def change
+    create_table :discourse_activity_pub_clients do |t|
+      t.integer :auth_type, null: false
+      t.string :domain, null: false
+      t.json :credentials, null: false
+
+      t.timestamps
+    end
+
+    execute <<~SQL
```

Comment:
This migration fails on my test instance with: 

```
2024-12-04 20:37:32.985 UTC [2062] discourse@discourse STATEMENT:  INSERT INTO discourse_activity_pub_clients(auth_type, domain, credentials)
	SELECT 2, key, value::json
	FROM plugin_store_rows
	WHERE plugin_name = 'discourse-activity-pub-oauth-app' AND value IS NOT NULL

rake aborted!
StandardError: An error has occurred, this and all later migrations canceled: (StandardError)

PG::NotNullViolation: ERROR:  null value in column "created_at" of relation "discourse_activity_pub_clients" violates not-null constraint
DETAIL:  Failing row contains (1, 2, mstdn.ca, {"client_id":"REDACTED","clie..., null, null).
```

Note that I had an existing authorization here to `mstdn.ca`.

Reply:
This is addressed: https://github.com/discourse/discourse-activity-pub/pull/133/commits/7622b671c2626c8ca7f7eb595decd66a67be0acd